### PR TITLE
Completed issue-1-create-an-entry

### PR DESCRIPTION
**Pull request by jeff-cruz**
Kapture 2022-04-04 at 20 29 16

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,11 +13,44 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link href="https://fonts.googleapis.com/css2?family=Proza+Libre:wght@400;600&display=swap" rel="stylesheet">
 </head>
-<body>
-  <header>
+<body class="row">
+  <header class="column-full">
+    <div class="container">
```
Comment (codingcodymiller) [inline index.html:20]: `container` should not be in a `column`, The order is always container -> row -> column -> row (if needed) -> column (if needed)

File: index.html [hunk 2]
```diff
@@ -13,11 +13,44 @@
...
+      <h2 class="font-proza">Code Journal</h2>
+    </div>
   </header>
-  <main>
+  <main class="container">
+    <h1 class="font-proza column-full">New Entry</h1>
```
Comment (codingcodymiller) [inline index.html:25]: `column` elements should never be direct children of a `container`. You need this `column` element to be a child of a `row` element.

File: js/main.js [hunk 3]
```diff
@@ -1,2 +1,33 @@
...
+  var formPhotoURL = document.querySelector('#photo-url').value;
+  var formNotes = document.querySelector('#notes').value;
+  var entry = {
+    title: formTitle,
+    photoURL: formPhotoURL,
+    notes: formNotes,
+    entryID: data.nextEntryId
+  };
+  data.nextEntryId++;
+  data.entries.push(entry);
```
Comment (codingcodymiller) [inline js/main.js:30]: You should be adding this object to the beginning of the array, not the end.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by jeff-cruz** [comment]
Kapture 2022-04-05 at 10 32 24

File: index.html [hunk 4]
```diff
@@ -15,41 +15,43 @@
...
   <link href="https://fonts.googleapis.com/css2?family=Proza+Libre:wght@400;600&display=swap" rel="stylesheet">
 </head>
-<body class="row">
-  <header class="column-full">
-    <div class="container">
-      <h2 class="font-proza">Code Journal</h2>
+<body>
+  <header>
+    <div>
+      <h2 class="font-proza container top-bottom-padding">Code Journal</h2>
```
Comment (codingcodymiller) [inline index.html:21]: You are still lacking a `row` and `column` element here. I would make this `container` have a child of a `row`, which has a child of a `column-full` that could hold the `h2` as a child.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Pull request merged**